code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -156,9 +153,9 @@
 
 	else //external -> internal
 		var/pressure_delta = 10000
-		if(vent_pump.pressure_checks & 1)
```

Comment:
Is `pressure_checks` meant to be a bitflag system or just use defines?

Reply:
Functionally they're currrently just `1` and `2`, defines are more accurate